core/datainput.gms

```diff
@@ -961,8 +958,11 @@ $if %cm_MAgPIE_coupling% == "on" p_efFossilFuelExtr(regi,"pebiolc","n2obio") = 0
 
 display p_efFossilFuelExtr;
 
-pm_dataren(regi,"nur",rlf,te)     = f_datarenglob("nur",rlf,te);
-pm_dataren(regi,"maxprod",rlf,te) = sm_EJ_2_TWa * f_datarenglob("maxprod",rlf,te);
+*** capacity factors (nur) are 1 by default
+pm_dataren(regi,"nur",rlf,te)     = 1;
+
+*** value copied from old file generisdata_nur_ren.prn (50EJ = 1.5855TWa)
```

Comment:
maybe even more explicitly: 
*** the only RE technology that doesn't get maxprod and nur from f_maxProdGradeRegixxx files are geohe and geohdr, which take their values from the old file generisdata_nur_ren.prn

Reply:
50EJ right?
Much better with these explanations thanks.